src/main/java/frc/robot/subsystems/Rollergripper.java

```diff
@@ -14,134 +14,134 @@
+            () -> {
+                rollers.set(0.75);
+                Shuffleboard.selectTab("armCamera");
+            },
+            () -> {
+                rollers.stopMotor();
+                Shuffleboard.selectTab("driveTab");
+            },
+            this)
           .until(beambreakTrigger);
-  }
-
-  /**
-   * ejectCommand
-   * <p><b> noInit </b>starts spinning the motors outwards<br>
-   * <b>noEnd </b>stops the motors</p>
-   * ends when the button senses that a cone is not longer in the roller gripper
-   * @return the command
-   */
-  public Command ejectCommand(double offset) {
-    return Commands.runEnd(
-                () -> {
-                  LeadRoller.set(-0.025 - offset);
-                },
-                () -> {
-                  LeadRoller.stopMotor();
-                },
-                this)
+    }
+
+    /**
+     * ejectCommand
+     * <p><b> noInit </b>starts spinning the motors outwards<br>
+     * <b>noEnd </b>stops the motors</p>
+     * ends when the button senses that a cone is not longer in the roller gripper
+     *
+     * @return the command
+     */
+    public Command ejectCommand(double offset) {
+        return Commands.runEnd(
+            () -> rollers.set(-0.025 - offset),
+            rollers::stopMotor,
+            this) //runEnd ends here
           .until(beambreakTrigger.negate().debounce(0.2));
-  }
-
-
-  public Command ejectCommand(){
-    return ejectCommand(0);
-  }
-
-  /**
-   * waits until the driver wants to release the cone and then returns the eject command
-   * @param release the button that needs to be pressed in order to eject
-   * @return the eject command after the button was pressed
-   */
-  public Command releaseCommand(BooleanSupplier release) {
-    return new RunCommand(() -> {}).until(release).andThen(ejectCommand());
-  }
-
-  /**
-   * applies a small force to the roller gripper in order to hold it in place
-   * @return the command
-   */
-  public Command holdConeCommand() {
-    return new ConditionalCommand(
-          setRollerGripperMotor(0.05).until(()-> true),
-          setRollerGripperMotor(0).until(()-> true),
+    }
+
+    public Command ejectCommand() {
+        return ejectCommand(0);
+    }
+
+    /**
+     * waits until the driver wants to release the cone and then returns the eject command
+     *
+     * @param release the button that needs to be pressed in order to eject
+     * @return the eject command after the button was pressed
+     */
+    public Command releaseCommand(BooleanSupplier release) {
+        return new WaitUntilCommand(release)
+          .andThen(ejectCommand());
+    }
+
+    /**
+     * applies a small force to the roller gripper in order to hold it in place
+     *
+     * @return the command
+     */
+    public Command holdConeCommand() {
+        return new ConditionalCommand(
+          setRollerGripperMotor(0.05).until(() -> true),
+          setRollerGripperMotor(0).until(() -> true),
```

Comment:
not really your fault but still something i want to change.. change both these lines from .until(()-> true) to .withTimeout(0.25)